ksql-engine/src/test/resources/query-validation-tests/elements.json

```diff
@@ -38,7 +38,8 @@
         },
         {
           "name": "S",
-          "format": "AVRO"
+          "format": "AVRO",
+          "partitions": 4
```

Comment:
this is fixing up tests that fail now that we have stricter checking of partition and rf counts even in the fake kafka topic client.